### PR TITLE
Adding Footer

### PR DESCRIPTION
**Pull request by Ferdinand-Ogama**
**Description**
-	I can’t find the documentation. Please direct me.
-	I have added the footer section on the website. The challenge I ran into was removing the horizontal strip between the footer and the bottom of the screen. I tried changing the screen view height, but it couldn’t work. 
-	This is what it looks like: 

Untitled

### CODE REVIEW

File: src/components/Footer/Footer.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            bgColor={"teal"}
+            width={"100%"}
+            p={4}
+            textAlign="center"
+            paddingTop={5}
+            paddingBottom={5}
+            color="#E7E6DA"
+        >
+            <Text fontSize="16" mb={2}>
+                SQL Online Editor - Copyright © {currentYear}
```
Comment (janvi01) [inline src/components/Footer/Footer.js:18]: Highlight the `janvi01` link, in spite of `a` tag you can use `link` tag

```suggestion
                SQL Online Editor - Copyright © {currentYear} | Created by <a href="https://github.com/janvi01" target="_blank">janvi01</a>
```

File: src/components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            </Text>
+            <HStack justifyContent="center" spacing={4}>
+                <Link href="https://github.com/janvi01" isExternal>
+                    <HStack>
+                        <BsGithub fontSize="1.5rem" />
+                    </HStack>
+                </Link>
```
Comment (janvi01) [inline src/components/Footer/Footer.js:25]:
```suggestion
```

File: src/pages/FrontLayout.js [hunk 3]
```diff
@@ -58,6 +60,7 @@ const FrontLayout = () => {
           setLoading={setLoading}
         />
       </Suspense>
+      <Footer />
```
Comment (janvi01) [inline src/pages/FrontLayout.js:63]: add the footer component in `App.js` not in `FrontLayout.js`

**Review by janvi01 — CHANGES_REQUESTED** [review]
LGTM 🚀 Kindly make the required changes @Ferdinand-Ogama Thanks!

**Comment by Ferdinand-Ogama** [comment]
Hello,  I have applied the changes.

**Review by janvi01 — APPROVED** [review]
LGTM @Ferdinand-Ogama I have made minor required changes, you can just go through what changes I have made in the recent commit. Also, use `prettier` to format code for this project from next time.

**Pull request merged**